source/crud/tests/server_2_6/read/aggregate.yml

```diff
@@ -0,0 +1,28 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+serverVersion: '2.6'
```

Comment:
Sounds good. Can we get the new `serverVersion` field documented in `crud/tests/README.rst`? Perhaps also add a one-liner to the main spec changelog to say that tests for version-specific functionality were also split out.